client-js/app/elements/labrad-plot.ts

```diff
@@ -783,7 +823,8 @@ export class Plot extends polymer.Base {
           x = this.mouseToDataX_(event.pageX - rect.left),
           y = this.mouseToDataY_(event.pageY - rect.top);
 
-    this.currPos = `(${prettyNumber(x, xMin, xMax, dx)}, ${prettyNumber(y, yMin, yMax, dy)})`;
+    this.currPos = `(${prettyNumber(x, xMin, xMax, dx)}, ` +
+                   `${prettyNumber(y, yMin, yMax, dy)})`;
```

Comment:
I'd suggest making these separate expressions to keep the string interpolation simple:
```typescript
const xStr = prettyNumber(x, xMin, xMax, dx),
      yStr = prettyNumber(y, yMin, yMax, dy);
this.currPos = `(${xStr}, ${yStr})`;
```